packages/schematics/angular/library/index.ts

```diff
@@ -61,42 +53,35 @@ function updateJsonFile<T>(host: Tree, path: string, callback: UpdateJsonFn<T>):
   return host;
 }
 
-function updateTsConfig(npmPackageName: string, entryFilePath: string) {
-
-  return (host: Tree) => {
-    if (!host.exists('tsconfig.json')) { return host; }
-
-    return updateJsonFile(host, 'tsconfig.json', (tsconfig: TsConfigPartialType) => {
-      tsconfig.compilerOptions.baseUrl = '.';
-      if (!tsconfig.compilerOptions.paths) {
-        tsconfig.compilerOptions.paths = {};
-      }
-      if (!tsconfig.compilerOptions.paths[npmPackageName]) {
-        tsconfig.compilerOptions.paths[npmPackageName] = [];
-      }
-      tsconfig.compilerOptions.paths[npmPackageName].push(entryFilePath);
-    });
-  };
-}
-
 function addDependenciesAndScriptsToPackageJson() {
 
   return (host: Tree) => {
     if (!host.exists('package.json')) { return host; }
 
     return updateJsonFile(host, 'package.json', (json: PackageJsonPartialType) => {
+      if (!json['dependencies']) {
+        json['dependencies'] = {};
+      }
+
+      json.dependencies = {
+        '@angular/common': '^5.2.0',
+        '@angular/core': '^5.2.0',
+        '@angular/compiler': '^5.0.0',
+        // De-structure last keeps existing user dependencies.
+        ...json.dependencies,
+      };
+
       if (!json['devDependencies']) {
         json['devDependencies'] = {};
       }
 
       json.devDependencies = {
-        '@angular/compiler': '^5.0.0',
         '@angular/compiler-cli': '^5.0.0',
         'ng-packagr': '^2.2.0',
         'tsickle': '>=0.25.5',
```

Comment:
`@angular/compiler-cli` has a direct dependency on `tsickle` is there a reason it is here as well?

Reply:
`ng-packagr` itself has a peer dependency on `tsickle`: https://github.com/dherges/ng-packagr/blob/master/package.json#L59-L64